index.js

```diff
@@ -4,15 +4,9 @@ const Router = require('koa-router');
 const uuid = require('node-uuid');
 const AWS = require('aws-sdk');
 
-const S3_ACCESS_KEY = process.env.AWS_ACCESS_KEY_ID;
-const S3_SECRET_ACCESS_KEY = process.env.AWS_SECRET_ACCESS_KEY;
```

Comment:
I don't see these being used at all anymore. The README still says setting environment variables is an option. I could merge, but I would update the README, then I'll do a major version bump